girder-dandi-archive/tests/rest/test_stats.py

```diff
@@ -125,7 +125,7 @@ def test_stats_cells(server, dandiset_1):
     } == resp.json
 
 
-def test_stats_size(server, fsAssetstore, user, dandiset_1):
+def test_stats_size(server, fsAssetstore, user, dandiset_1):  # NOQA
```

Comment:
```suggestion
def test_stats_size(server, fsAssetstore, user, dandiset_1):  # noqa: N803
```

Should be lowercase: http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors
Should be as specific to a single error as possible, so other errors on that line aren't silenced.